kernel/kernel-api/src/test/scala/io/delta/kernel/internal/util/SchemaUtilsSuite.scala

```diff
@@ -316,8 +387,16 @@ class SchemaUtilsSuite extends AnyFunSuite {
       // no issues here
       Seq(s"a${char}b", s"${char}ab", s"ab${char}", char.toString).foreach { name =>
         val schema = new StructType().add(name, INTEGER);
-        validateSchema(schema, false /* isColumnMappingEnabled */ )
-        validateSchema(schema, true /* isColumnMappingEnabled */ )
+        validateSchema(
+          schema,
+          false,
+          false,
+          false)
+        validateSchema(
+          schema,
+          true,
+          false,
+          false)
       }
     }
```

Comment:
Can we add a simple defaultColumns test case here? basically to verify that `ColumnDefaults. validateSchema` is called as part of the SchemaUtils.validateSchema. Can be super simple.